web/app/scripts/workspace/html-state.html

```diff
@@ -0,0 +1,6 @@
+<uib-progressbar ng-if="!content.$resolved" value="100" class="progress-striped active">
+Loading...
+</uib-progressbar>
+<div class="col-sm-12" ng-show="content.$resolved">
+  <div trusted-html="content.string"></div>
+</div>
```

Comment:
Add a newline please.

Reply:
> Add a newline please.

Weird. The missing newline was fixed 20 days ago. Did you have this browser tab open that long? 😄  Here's the commit history of this file: https://github.com/lynxkite/lynxkite/commits/darabos-html/web/app/scripts/workspace/html-state.html

Anyway, the newline is good now. 👍 Thanks for the review!